pkg/daemon/ceph/client/mgr.go

```diff
@@ -19,15 +19,34 @@ package client
 import (
 	"fmt"
 	"strings"
+	"time"
 
 	"github.com/pkg/errors"
 	"github.com/rook/rook/pkg/clusterd"
 	cephver "github.com/rook/rook/pkg/operator/ceph/version"
 )
 
+var (
+	moduleEnableWaitTime = 5 * time.Second
+)
+
 // MgrEnableModule enables a mgr module
 func MgrEnableModule(context *clusterd.Context, clusterName, name string, force bool) error {
-	return enableModule(context, clusterName, name, force, "enable")
+	retryCount := 5
+	for i := 0; i < retryCount; i++ {
+		err := enableModule(context, clusterName, name, force, "enable")
+		if err != nil {
+			if i < retryCount-1 {
+				logger.Infof("failed to enable mgr module %q. trying again...", name)
```

Comment:
I'd vote for this to at least be a Warning instead.

Reply:
Similar retry logs have been `info`. What do yo say?  https://github.com/rook/rook/blob/cfa34d29732ab1fe2fc8bd9530c504f98eccdf3f/pkg/operator/ceph/cluster/mgr/dashboard.go#L207

Reply:
Updated to `warning`